### PR TITLE
NPC Sprites

### PR DESCRIPTION
**Pull request by nanaian**
A header file for each sprite is created in `include/sprite`. I originally intended to make an aggregate file with all animation ids etc (similar to `enums.h`), but this forced every single map C file to have to rebuild whenever you changed a sprite palette name or something. Instead, discrete headers for each sprite are generated and maps include what they use (e.g. `kmr_12/events.c` includes `sprite/npc/goomba.h` directly).

### CODE REVIEW

**Review by ethteck — APPROVED** [review]

File: tools/splat.yaml
Comment (pixel-stuck) [inline tools/splat.yaml:7031]: does n64splat properly pad this? Given the address used to be 0x19E67B2 and is now 0x19E67B8; even if it does, maybe we should have some kind of ALIGN thing in splat? Not sure. Second part is a bit out of scope for this PR, though.

**Review by pixel-stuck — APPROVED** [review]
what's up with tools/sort_symbol_addrs.py? Github is showing it as an empty file. After merge conflicts are resolved, lgtm.

File: tools/splat.yaml
Comment (ethteck) [reply tools/splat.yaml:7031]: oh yipes. I actually am not sure... I mean there's no code to manually handle alignment of sections or anything, but there probably should be..

Comment (nanaian) [reply tools/splat.yaml:7031]: Well, the build matches... this will be resolved once I get around to implementing player sprites and the sprite table header (at this address IIRC) in a future PR.

**Comment by nanaian** [comment]
@pixel-stuck I made `tools/sort_symbol_addrs.py` executable

**Comment by ethteck** [comment]
will merge once conflict is resolved. (sorry for the ugly conflict)

**Pull request merged**
